defaults/main.yml

```diff
@@ -5,6 +5,7 @@ fluentbit_prerequisites:
   - curl
   - gnupg
 
+fluentbit_groups: "adm"
```

Comment:
fluentbit_groups is a list, isn't it ?

Reply:
It a string separated by `,`, better to make a list and a join!